.changeset/dirty-taxis-learn.md

```diff
@@ -0,0 +1,7 @@
+---
+"vite-env-only": minor
+---
+
+Allow globs for `denyImports` and `denyFiles`
+
+Using [micromatch][https://github.com/micromatch/micromatch] for pattern matching
```

Comment:
Out of curiosity, why'd you go with this library?

Reply:
Vite uses `fast-glob`, and `fast-glob` uses `minimatch`

Reply:
> `fast-glob` uses `micromatch`

Ah nice, I was missing this context 👍